engine/exec/exec.go

```diff
@@ -19,10 +20,10 @@ func Run(ctx context.Context, step *spec.Step, output io.Writer) (*runtime.State
 		return nil, errors.New("step entrypoint cannot be empty")
 	}
 
-	cmdArgs := step.Entrypoint[1:]
-	cmdArgs = append(cmdArgs, step.Command...)
+	// use runner-go shell script logic to generate the script
```

Comment:
there would be different behaviour for drone vs cie here. 
with drone we will have already wrapped the commands in a shell script by the time it reaches here. So it would be double wrapped at this point.  a bash script calling another bash script to execute commands.